internal/codegen/golang/field.go

```diff
@@ -77,3 +77,12 @@ func toCamelInitCase(name string, initUpper bool) string {
 	}
 	return out
 }
+
+func toLowerCase(str string) string {
+	var b strings.Builder
```

Comment:
Note that this panics on an empty string. And it is over complicated. 

I suggest:

```go
if str == "" {
  return ""
}
return strings.ToLower(string[:1]) + string[1:]
```